src/tools/gophish_export.py

```diff
@@ -253,6 +254,71 @@ def get_application(rawEvent):
     return application
 
 
+def find_unique_target_clicks_count(clicks):
+    """Return the number of unique clicks in a click dict."""
+    uniq_list = list()
+    for click in clicks:
+        if click["user"] not in uniq_list:
+            uniq_list.append(click["user"])
+    return len(uniq_list)
+
+
+def write_assessment_click_summary(api, assessment_id):
+    """Output a click summary report to JSON, console, and a text file."""
+    assessment_click_summary = dict()
+    campaign_click_stats = dict()
+    campaigns = list()
+    campaign_ids = get_campaign_ids(api, assessment_id)
+    num_campaigns = len(campaign_ids)
+    json_filename = assessment_id + "_click_summary.json"
+    summary_outfile_name = assessment_id + "_click_summary.txt"
+    if os.path.exists(summary_outfile_name):
+        os.remove(summary_outfile_name)
+    fh = logging.FileHandler(summary_outfile_name, "w+")
```

Comment:
If `summary_outfile_name` exists and you remove it, then there should no longer be any need for "w+" mode.
```suggestion
    if os.path.exists(summary_outfile_name):
        os.remove(summary_outfile_name)
    fh = logging.FileHandler(summary_outfile_name, "w")
```

Reply:
Looks like this was corrected back in https://github.com/cisagov/gophish-tools/commit/0a0c1649c0f6a2b3debf21bc8f1682c4eb37a8d5#diff-56cd9a15b1d8135c0ca3f40187af79b1c273eed9a40dbc3c8f6d43705925674bL275-L276 - thanks!